### PR TITLE
remove event-bus subtest from upgrade test

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- remove event-bus test

**Related issue(s)**

#7731

### CODE REVIEW

File: tests/end-to-end/upgrade/chart/upgrade/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ containerRegistry:
 
 image:
   dir:
-  version: 55ccf207
+  version: PR-7755
```
Comment (marcobebway) [inline tests/end-to-end/upgrade/chart/upgrade/values.yaml]: Is this for verification purposes?

As this change should be in another PR with the final image SHA.

Comment (k15r) [reply tests/end-to-end/upgrade/chart/upgrade/values.yaml]: yes that will happen ... but as all those updated tests create new images i want to wait until all evidence of event-bus has been wiped from the test folder and then create one pr to update all of them
